style.css

```diff
@@ -1 +1,172 @@
 /*  Add your styling here */
+body {
+  font: 1em sans-serif;
+  align-content: center;
+  align-items: center;
+}
+
+header {
+  width: 95%;
+  margin-left: auto;
+  margin-right: auto;
+  height: auto;
+  padding-top: 2%;
+  padding-bottom: 2%;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  background-color: rgb(253, 236, 239);
+
+  border: 3px darkblue solid;
+}
+
+header img {
+  width: 50px;
+  height: 50px;
+
+  border: 1px red solid;
+}
+
+header div {
+  width: 60%;
+  display: table;
+
+  border: 1px red solid;
+}
+
+header div h4 {
+  width: auto;
+  text-align: right;
+  opacity: 0.6;
+  font-style: italic;
+  margin: 0;
+
+  border: 1px red solid;
+}
+header nav {
+  text-align: right;
+
+  border: 1px red solid;
+}
+
+#welcome {
+  width: 95%;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  border: darkred 3px solid;
+}
+
+#welcome .welcome-text {
+  width: auto;
+  height: 50%;
+  object-fit: contain;
+
+  display: block;
+  object-fit: cover;
+  justify-content: flex-start;
+  padding: 5%;
+  border: 1px red solid;
+}
+
+#welcome .welcome-image {
+  width: 100%;
+  height: 50%;
+  object-fit: cover;
+
+  display: block;
+  border: 1px red solid;
+}
+
+img {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: cover;
```

Comment:
This works here, but remember what we saw in class: `object-fit` does not work on all browsers, so it's best to learn how to style images without this.

Reply:
Don't set a width AND a height on your `img` as this will distort them. `width: 100%` should be enough, and let the height set itself naturally.

Reply:
fixed it